dune-project

```diff
@@ -146,20 +146,6 @@ semi-interactive development of machine-checked proofs.
 This package provides the RocqIDE, a graphical user interface for the
 development of interactive proofs."))
 
-(package
- (name rocq-doc)
- (license "OPUBL-1.0")
- (depends
-  (conf-python-3 :build)
-  (coq (and :build (= :version))))
```

Comment:
This part used to give the dependency of the documentation targets on `coq`, so I assume this is the cause of #20825.